platform/windows/os_windows.cpp

```diff
@@ -953,7 +953,7 @@ String OS_Windows::get_user_data_dir() const {
 String OS_Windows::get_unique_id() const {
 	HW_PROFILE_INFO HwProfInfo;
 	ERR_FAIL_COND_V(!GetCurrentHwProfile(&HwProfInfo), "");
-	return String::utf16((const char16_t *)(HwProfInfo.szHwProfileGuid), HW_PROFILE_GUIDLEN);
+	return String((HwProfInfo.szHwProfileGuid), HW_PROFILE_GUIDLEN);
```

Comment:
```c++
	HW_PROFILE_INFOA HwProfInfo;
	ERR_FAIL_COND_V(!GetCurrentHwProfileA(&HwProfInfo), "");
	return String((HwProfInfo.szHwProfileGuid), HW_PROFILE_GUIDLEN);
```

Probably better to explicitly specify it's an ASCII version to avoid any future mistakes. (or use `W` versions and `::utf16(...)`, but in this case it's GUID, so extra conversion is not necessary).

Reply:
Amended the commit as suggested here.